### PR TITLE
feat(ui5-color-palette): implement mobile view

### PR DESCRIPTION
**Pull request by tsanislavgatev**

### CODE REVIEW

File: packages/main/src/ColorPalette.ts [hunk 1]
```diff
@@ -153,6 +153,13 @@ class ColorPalette extends UI5Element {
 	@property({ type: Boolean })
 	popupMode!: boolean;
 
+	/**
+	 * Defines if the palette is rendered on phone.
+	 * @private
+	 */
+	@property({ type: Boolean })
+	onPhone!: boolean;
```
Comment (nnaydenow) [inline packages/main/src/ColorPalette.ts:161]: rename that one on phone or the other one placed in ColorPaletteItem for consistency

File: packages/main/src/ColorPalettePopover.hbs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 >
 	<div slot="header" class="ui5-cp-header">
 		<ui5-title
-			class="ui5-cp-title"
+			level="H5"
```
Comment (nnaydenow) [inline packages/main/src/ColorPalettePopover.hbs:11]: are we sure that the heading level should be h5? can we guarantee that the headings map will be correct?

File: packages/main/src/ColorPalettePopover.hbs [hunk 3]
```diff
@@ -25,10 +25,11 @@
 			{{/each}}
 		</ui5-color-palette>
 	</div>
-	<div slot="footer" class="ui5-cp-footer">
+	<bar slot="footer" class="ui5-cp-footer">
```
Comment (nnaydenow) [inline packages/main/src/ColorPalettePopover.hbs:28]: why do we change div to bar? bar is not a valid html element at all

**Review by nnaydenow — CHANGES_REQUESTED** [review]
overall these percents in the css are looking strange for me and I would remove them

File: packages/main/src/ColorPalettePopover.hbs (hunk 2 above)
Comment (tsanislavgatev) [reply packages/main/src/ColorPalettePopover.hbs:11]: Yes, it's by design.

**Comment by tsanislavgatev** [comment]
> overall these percents in the css are looking strange for me and I would remove them

There is no exact width or height for neither the items nor the containers. It should be responsible and depend on eachother. Any suggestions are welcomed.

**Review by Todor-ads — APPROVED** [review]

File: packages/main/src/ColorPalette.ts (hunk 1 above)
Comment (tsanislavgatev) [reply packages/main/src/ColorPalette.ts:161]: I was planning on doing it in separate change so I don't mix it up with this one, but i can do it in this one as well.

**Review by nnaydenow — APPROVED** [review]

**Comment by tsanislavgatev** [comment]
fixes: https://github.com/SAP/ui5-webcomponents/issues/3381

**Pull request merged**